platform-controller/src/main/java/org/hobbit/controller/ExperimentManager.java

```diff
@@ -146,9 +150,17 @@ public void createNextExperiment() {
             return;
         }
         try {
-            // if there is no benchmark running (and the queue has been
-            // initialized)
-            if ((experimentStatus == null) && (controller.queue != null)) {
+            // if there is no benchmark running, the queue has been
+            // initialized and cluster is healthy
+            ClusterManager clusterManager = new ClusterManagerImpl();
+            boolean isClusterHealthy = clusterManager.isClusterHealthy();
+            if(!isClusterHealthy) {
+                LOGGER.error("Can not start next experiment in the queue, cluster is NOT HEALTHY. " +
+                        "Check your cluster consistency or adjust SWARM_NODE_NUMBER environment variable." +
+                        "Expected number of nodes: "+clusterManager.getExpectedNumberOfNodes()+
+                        "Current number of nodes: "+clusterManager.getNumberOfNodes());
```

Comment:
tiny issue: 
```
"Check your cluster consistency or adjust SWARM_NODE_NUMBER environment variable." +
"Expected number of nodes: "+clusterManager.getExpectedNumberOfNodes()+
"Current number of nodes: "+clusterManager.getNumberOfNodes());
```
will lead to
```
"Check ... variable.Expected number of nodes: 6Current number of nodes: 5
```
You may want to add whitespaces before "Expected" and "Current" :wink: